File: PROYECTO/PSport-APP/PSport/Model/DTOs/PistaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class PistaDTO
    {
        public Guid id { get; set; }
        public string nombre { get; set; }
        public string tipoPista { get; set; }
        public string nombrePolideportivo { get; set; }
        public string polideportivoID { get; set; }

    }
}

File: PROYECTO/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/DTO/PistaDTO.kt
package es.sergigavi.PPSportAPI.MODEL.DTO

import es.sergigavi.PPSportAPI.MODEL.Deporte
import java.util.*

data class PistaDTO(
    var id: UUID,
    var nombre:String,
    var tipoPista: Deporte,
    var nombrePolideportivo:String,
    var polideportivoID: UUID,
)
